stories/module-pagespeed-insights-components.stories.js

```diff
@@ -31,14 +31,59 @@ import { STORE_NAME as CORE_SITE } from '../assets/js/googlesitekit/datastore/si
 import * as fixtures from '../assets/js/modules/pagespeed-insights/datastore/__fixtures__';
 import { STRATEGY_MOBILE, STRATEGY_DESKTOP } from '../assets/js/modules/pagespeed-insights/datastore/constants';
 import { WithTestRegistry } from '../tests/js/utils';
+import fetchMock from 'fetch-mock';
 
 storiesOf( 'PageSpeed Insights Module/Components', module )
 	.add( 'Dashboard widget', () => {
 		const url = fixtures.pagespeedMobile.loadingExperience.id;
 		const setupRegistry = ( { dispatch } ) => {
 			dispatch( STORE_NAME ).receiveGetReport( fixtures.pagespeedMobile, { url, strategy: STRATEGY_MOBILE } );
 			dispatch( STORE_NAME ).receiveGetReport( fixtures.pagespeedDesktop, { url, strategy: STRATEGY_DESKTOP } );
-			dispatch( CORE_SITE ).receiveSiteInfo( { referenceSiteURL: url } );
+			dispatch( CORE_SITE ).receiveSiteInfo( {
+				referenceSiteURL: url,
+				currentEntityURL: null,
+			} );
+		};
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<Layout>
+					<DashboardPageSpeed />
+				</Layout>
+			</WithTestRegistry>
+		);
+	} )
+	.add( 'Dashboard widget (loading)', () => {
+		fetchMock.getOnce(
+			/^\/google-site-kit\/v1\/modules\/pagespeed-insights\/data\/pagespeed/,
+			new Promise( () => {} ) // Never returns a response for perpetual loading.
```

Comment:
This comment is helpful but took me a second to parse. I think we could clarify it a bit:

```suggestion
			new Promise( () => {} ) // Never return a response, to keep the component in a perpetual "loading" state.
```